hogvm/python/stl/date.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from typing import Optional
+
+import pytz
+
+
+def now():
+    return datetime.datetime.now()
+
+
+def toUnixTimestamp(date, timezone: Optional[str] = None):
+    if isinstance(date, datetime.date) and not isinstance(date, datetime.datetime):
+        date = datetime.datetime(date.year, date.month, date.day, tzinfo=pytz.timezone(timezone or "UTC"))
+    if isinstance(date, str):
+        date = datetime.datetime.fromisoformat(date)
+        if timezone:
+            date = date.astimezone(pytz.timezone(timezone))
+    return date.timestamp()
+
+
+def fromUnixTimestamp(timestamp):
+    return datetime.datetime.fromtimestamp(timestamp, tz=datetime.UTC)
+
+
+def toTimeZone(date, timezone: str):
+    return date.astimezone(pytz.timezone(timezone))
+
+
+def toDate(string):
+    return datetime.date.fromisoformat(string)
+
+
+def toDateTime(string):
```

Comment:
The other function that I could imagine being _most_ common would be something like
```
toDateTime(x.timestamp).format("MM-dd-YY") // Whatever custom format the destination requires
```

Not a biggy that we don't have it but would be interesting to consider, especially given that we are providing two different datetime implementations under the hood...

Reply:
Yeah, format will come very soon. Obviously ClickHouse, Python and NodeJS implement it very differently... so I decided to leave that for another PR.